src/neuroconv/datainterfaces/icephys/baseicephysinterface.py

```diff
@@ -33,6 +25,16 @@ def get_source_schema(cls) -> dict:
         return source_schema
 
     def __init__(self, file_paths: list):
+
+        try:
+            from ndx_dandi_icephys import DandiIcephysMetadata
```

Comment:
While we're here, can we replace the antiquated try/except block with `if < importlib? >.find_spec`?

Reply:
I am lazy to do it but is the right way.

Reply:
OK, changed to an importlib findspec based solution.